Description: Disable manpage installation in Makefile
 The manpage installation is done by dh_installmanpages
 so the installation in the Makefile is not needed
Author: Jan Luca Naumann <j.naumann@fu-berlin.de>
Last-Update: 2016-06-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -55,7 +55,7 @@
 	$(INSTALL_DATA) init/psd-resync.service "$(DESTDIR)$(INITDIR_SYSTEMD)/psd-resync.service"
 	$(INSTALL_DATA) init/psd-resync.timer "$(DESTDIR)$(INITDIR_SYSTEMD)/psd-resync.timer"
 
-install: install-bin install-man install-systemd
+install: install-bin install-systemd
 
 uninstall-bin:
 	$(RM) "$(DESTDIR)$(BINDIR)/$(PN)"
